packages/DateTimePickerCommon/src/datePickerStyles.ts

```diff
@@ -302,9 +302,12 @@ export const datePickerStyles = css`
 
   .react-datepicker__month .react-datepicker__month-text,
   .react-datepicker__month .react-datepicker__quarter-text {
-    display: inline-block;
+    display: inline-flex;
     width: 4rem;
+    height: 2rem;
```

Comment:
I did this to improve the style when we display months:

### Before
![Screenshot 2023-11-27 at 17 53 37](https://github.com/WTTJ/welcome-ui/assets/1446004/6f9db396-8b23-4e7f-993a-a32eb4d56750)

### After
![Screenshot 2023-11-27 at 17 53 19](https://github.com/WTTJ/welcome-ui/assets/1446004/42ce64ae-a06c-48ee-ad99-e80dcf922bd8)